cli/src/cli-quotad-client.c

```diff
@@ -34,16 +34,14 @@ cli_quotad_submit_request(void *req, call_frame_t *frame, rpc_clnt_prog_t *prog,
         };
 
         if (!iobref) {
-            iobref = iobref_new();
+            iobref = add_iobuf_to_new_iobref(iobuf);
             if (!iobref) {
                 goto out;
             }
 
             new_iobref = 1;
         }
```

Comment:
You are missing an else part - if there's already an iobref, you need to have the iobref_add() function.

Reply:
Done